ehrql/tables/beta/tpp.py

```diff
@@ -628,10 +628,37 @@ class sgss_covid_all_tests(EventFrame):
 
 @table
 class vaccinations(EventFrame):
-    vaccination_id = Series(int)
-    date = Series(datetime.date)
-    target_disease = Series(str)
-    product_name = Series(str)
+    """
+    Vaccinations
+
```

Comment:
Don't forget to check [the rendered docs](https://milanwiedemann-document-vacc.databuilder.pages.dev/reference/schemas/beta.tpp/#vaccinations). This "Vaccinations" doesn't contribute anything.

```suggestion
```